tests/Pest/FaviconTest.php

```diff
@@ -1,14 +1,27 @@
 <?php
 
-use function PHPUnit\Framework\assertFileExists;
+use ArchTech\SEO\Commands\GenerateFaviconsCommand;
 
-test("it should throw an exception if the given source icon doesn't exist", function () {
-    seo()->favicon('i-dont-exist.png');
-})->throws(Exception::class, 'Given icon path `i-dont-exist.png` does not exist.');
+use function Pest\Laravel\artisan;
+use function PHPUnit\Framework\assertFileExists;
 
 test('it should generate two favicons', function () {
-    seo()->favicon(__DIR__ . '/../stubs/logo.png');
+    seo()->favicon();
+
+    $from = __DIR__ . '/../stubs/logo.png';
+
+    artisan(GenerateFaviconsCommand::class, [
+        'from' => $from,
+    ])->assertSuccessful();
 
     assertFileExists(public_path('favicon.ico'));
     assertFileExists(public_path('favicon.png'));
 });
+
+test('it should fail because the from path is incorrect', function () {
+    seo()->favicon();
+
+    artisan(GenerateFaviconsCommand::class, [
+        'from' => 'i/dont/exist.png',
+    ])->assertFailed();
```

Comment:
Add a `beforeEach()` to clean `public_path()` favicons, and then assert after this check that the files don't exist.

And then we can merge 👍🏻